### PR TITLE
Revert added type hints

### PR DESCRIPTION
**Pull request by wbloszyk**
## Subject

This PR revert type hints added in docblock which should not be added. They will be add corectly in other PR like https://github.com/sonata-project/SonataUserBundle/pull/1198.

I am targeting this branch, because it is FIX before release.

Changelog is not needed becouse this is fix for non release PR.

### CODE REVIEW

**Comment by wbloszyk** [comment]
@phansys @VincentLanglet @core23 @jordisala1991 @greg0ire Can you merge this PR? It is last commit require to release MediaBundle. It will be nice to have it before tomorrow work.

File: src/Controller/Api/GalleryController.php [hunk 1]
```diff
@@ -339,8 +339,8 @@ public function putGalleryMediaGalleryhasmediaAction($galleryId, $mediaId, Reque
      *  }
      * )
      *
-     * @param int $galleryId A gallery identifier
-     * @param int $mediaId   A media identifier
+     * @param int $galleryId Gallery identifier
```
Comment (jordisala1991) [inline src/Controller/Api/GalleryController.php:342]: remove int?

Comment (wbloszyk) [reply src/Controller/Api/GalleryController.php:342]: I only revert change from https://github.com/sonata-project/SonataMediaBundle/pull/1767.

Comment (jordisala1991) [reply src/Controller/Api/GalleryController.php:342]: I think we should at least be consistent on this file, are we leaving the int or removing it? And the same for the property description, If we are leaving it, dont remove it for some cases

Comment (wbloszyk) [reply src/Controller/Api/GalleryController.php:342]: Some bundles must be release one by one to allow use `FOS/RestBundle` becouse of dependencies. Upgrade(and fix) RestApi consists from spets like add `fos/rest-bundle` v3 support,  `nelmio/api-doc-bundle` v3 support, add support for string identifier (like UUID), functional test. It is hard to do it in one PR(and shouldn't be do it). In this release we should add support for `fos/rest-bundle` v3 and do nothing else (like add, fix or remove other docblock). I mean this should be trait as two indepentend things. The goal will be support `RestBundle` and `ApiDocBundle` v2 and v3, and support for diffrence identifier(int, UUID, string ....). I know it is not good but keep it not working it not too.

**Comment by wbloszyk** [comment]
Docblocks will be add in diffrence PR with some change. For now only integer (like ApiDoc annotation require) is allowed. Technicly identifier can be UUID or string and this will also be change. We should not release docblock type hints and change in in other next release.

**Comment by wbloszyk** [comment]
@jordisala1991 Can we merge it?

**Comment by jordisala1991** [comment]
Yes sure

**Pull request merged**